operations/experimental/README.md

```diff
@@ -9,12 +9,113 @@ This is the README for Experimental Ops-files. To learn more about `cf-deploymen
 - For Community Ops-files, check out the [Community Ops-file README](../community/README.md).
 - For Legacy Ops-files, checkout the [Legacy Ops-file README](../legacy/README.md).
 
-"Experimental" ops-files represent configurations
-that we expect to promote to blessed configuration eventually,
-meaning that,
-once the configurations have been sufficiently validated,
-they will become part of cf-deployment.yml
-and the ops-files will be removed.
+
+# Promoting/Deprecating experimental ops files
```

Comment:
The heading:
"Promoting/Deprecating experimental ops files"

much of this process can be applied to any ops file regardless of whether it's in experimental or in the parent operations directory, or another sub-directory of operations (for example: backup-and-restore or bits-service).

As such, might be better to update heading to 

"Promoting/Deprecating ops files"